about-me.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>First Page</title>
+    <link rel="stylesheet" href="index.css">
+
+</head>
+
+<body style = background:rgb(27,38,59)>
+<header class="header">
+    <p id="myheader">Ed's Kitchen </p>
+    
+     <nav>
+            <a class="nav_item" href="index.html"> Home</a>
+            <a class="nav_item" href =about-me.html >About me </a>
+     </nav>
+</header>
+
+<main>
+    <div class="card" > 
+        <p>I cannot cook for the life of me. Take all of the information provided(especially the steps)
+            with a grain of salt. </p>
```

Comment:
This p element could use a left margin/padding. If you look at your about-me page the text is right on the edge of the element there which isn't the easiest to read